### PR TITLE
Update community.yml

### LINKED ISSUE #11: Update Community.yml
I will update the issue-message, Pr-Message, with new ones and add a footer message in Community.yml

### PR DESCRIPTION
**Pull request by muthuannamalai12**
Fixes #11 

## Proposed Changes

  - I have updated the issue-message, Pr-Message, with new ones and added a footer message in Community.yml

### CODE REVIEW

File: .github/workflows/community.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 on:
   fork:
   push:
-    branches: [ main ]
+    branches: [ develop ]
```
Comment (eddiejaoude) [inline .github/workflows/community.yml]: This branch does not exist on this repo. I do not think we need a `develop` branch on this repo as there is not much activity

File: .github/workflows/community.yml [hunk 2]
```diff
@@ -18,11 +17,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: EddieHubCommunity/gh-action-community/src/welcome@main
+      - uses: EddieJaoudeCommunity/gh-action-community/src/welcome@main
```
Comment (eddiejaoude) [inline .github/workflows/community.yml]: This was originally correct with `EddieHubCommunity`

**Comment by eddiejaoude** [comment]
Thank you 👍 , I left 1 small comment 🤓

**Review by AllanRegush — APPROVED** [review]
LGTM 👍

**Comment by eddiejaoude** [comment]
Thank you both! Sorry just catching up with GitHub notifications now - I am 10 days behind 🤦‍♂️

**Pull request merged**